### PR TITLE
Fixes to get the message-update branch work with LPDAAC deployment

### PR DESCRIPTION
**Pull request by scisco**

### CODE REVIEW

File: packages/ingest/granule.js [hunk 1]
```diff
@@ -146,31 +146,42 @@ class Discover {
...
+      this.stack,
+      this.bucket
+     )));
   }
 }
 
 
 /**
- * This is a base class for ingesting and parsing a single PDR
+ * This is a bse class for ingesting and parsing a single PDR
```
Comment (laurenfrederick) [inline packages/ingest/granule.js]: Did you mean to change this to bse?

File: packages/ingest/queue.js [hunk 2]
```diff
@@ -12,70 +11,72 @@ const {
...
-
-  message.input = { pdr };
+  * Create a message from a template stored on S3
+  *
+  * @param {string} queueUrl - The SQS url
+  * @param {string} templateUri - S3 uri to the workflow template
+  * @param {Object} provider - Cumulus provider object
+  * @param {Object} collection - Cumulus collection object
+  * @param {Object} pdr - the PDR object
+  * @param {string} pdr.name - name of the PDR
```
Comment (laurenfrederick) [inline packages/ingest/queue.js]: Should we be listing pdr.name if it's not a separate param?

File: packages/deployment/app/kes.override.js [hunk 3]
```diff
@@ -23,127 +116,183 @@ function generateKeyPair() {
...
 
-function findOutputValue(outputs, key) {
-  return outputs.find((o) => (o.OutputKey === key)).OutputValue;
+/**
+ * Genrates private/public keys and Upload them to a given bucket
+ *
+ * @param {string} bucket - the bucket to upload the keys to
+ * @param {string} key - the key (folder) to use for the uploaded files
+ * @param {Object} s3 - an instance of the AWS S3 class
+ * @returns {Promise} undefined
```
[the remark this answers is not in the document]
Comment (scisco) [reply packages/deployment/app/kes.override.js]: `async` functions always return a promise even if there is no actual return in the function.

File: packages/deployment/app/kes.override.js [hunk 4]
```diff
@@ -23,127 +116,183 @@ function generateKeyPair() {
...
+  // add the s3 uri to all the workflow templates for teh current stack
+  const templatesUris = {};
+  Object.keys(config.stepFunctions).forEach((sf) => {
+    templatesUris[sf] = `s3://${config.buckets.internal}/${config.stack}/workflows/${sf}.json`;
   });
 
-  // update cumulus_meta for each workflow message tempalte
-  template.cumulus_meta.state_machine = wfArn;
-  template.cumulus_meta.workflow_name = sf.name;
+  const t = {
```
[the remark this answers is not in the document]
Comment (scisco) [reply packages/deployment/app/kes.override.js]: not sure what you mean here. This function generate the Cumulus Message template for each workflow.

File: packages/ingest/queue.js (hunk 2 above)
Comment (scisco) [reply packages/ingest/queue.js]: this is a jsdoc's feature where you can basically tell people what properties in the pdr object you expect. This helps the developers to to know the pdr object must have a property key called name.

**Pull request merged**